src/pages/Login/Login.js

```diff
@@ -1,8 +1,22 @@
 import React from 'react';
+import KakaoLogin from './KakaoLogin/KakaoLogin';
+
 import './Login.scss';
 
 const Login = () => {
-  return <div>Login</div>;
+  return (
+    <div className="Login">
+      <div className="loginWrapper">
+        <KakaoLogin />
+      </div>
+      <div className="serviceInfoContainer">
+        <img
+          className="onebookLogo"
+          src="../images/OneBook_Logo.png"
```

Comment:
public 폴더에 있는 이미지 경로는, 상대경로가 아닌 절대경로로 작성하시면 됩니다.
```suggestion
          src="/images/OneBook_Logo.png"
```